.gitignore

```diff
@@ -1,3 +1,7 @@
-*.wpr
-*.wpu
-.*pyc
\ No newline at end of file
+# Ignore python byte code.
+*.pyc
+*~
+
+# Ignore editor configurations.
+.project
+.pydevproject
```

Comment:
Definitely... 

Sorry. Originally I started editing this thinking we'd only be using local changes in the office.